share/smack/doctor.py

```diff
@@ -129,7 +128,7 @@ def main():
     check_command("llvm2bpl")
     check_command("smack")
 
-    if args.prefix is not '':
+    if args.prefix != '':
```

Comment:
Let's rewrite this as `if not args.prefix` since that is it seems that pep8 prescribes:
https://stackoverflow.com/questions/9573244/how-to-check-if-the-string-is-empty